lib/helpers.py

```diff
@@ -0,0 +1,43 @@
+import typing
+
+import string
+from helpers import query as orig_query, update as orig_update, log, generate_uuid
+from escape_helpers import sparql_escape_uri, sparql_escape_string
+from .. import config
+from . import exceptions
+
+class Template():
```

Comment:
Dit vind ik geen goede abstractie. Voor derden is dit verwarrend om lezen. Het gaat om een klassenaam die in de standaard-modules al beschikbaar is, die hier gewrapped wordt in functie van enkele booleaanse testen (die ongetwijfeld niet sluitend zijn) om op het moment van substitutie een gok te maken naar of dit al dan niet een valide query zal zijn.  
Ik begrijp van waar je komt. Je wil hier eigenlijk een soort van development tool invoegen die je vroegtijdig waarschuwt voor een slecht samengestelde query. Dit zou iets kunnen zijn voor een toolbox/ide of dergelijke, maar niet voor productie-code. Graag de custom `Template` klasse verwijderen en overal de standaard `Template` een `subsitute` gebruiken.  
Indien je nood hebt aan meer doorgedreven validatie-tooling, komen we hier later nog eens op terug.